fastlane/lib/fastlane/actions/slather.rb

```diff
@@ -29,7 +29,9 @@ class SlatherAction < Action
           configuration: '--configuration',
           workspace: '--workspace',
           binary_file: '--binary-file',
+          binary_files: '--binary-file',
           binary_basename: '--binary-basename',
+          binary_basenames: '--binary-basename',
```

Comment:
I am not sure how this works? Do the long flag names have to be different than the singular case?